ec/ecresource/deploymentresource/elasticsearchstate/elasticsearch_flatteners.go

```diff
@@ -71,8 +72,9 @@ func FlattenResources(in []*models.ElasticsearchResourceInfo, name string) []int
 			m[k] = v
 		}
 
-		// TODO: Flatten repository state.
-		// Determine what to do with the default snapshot repository.
```

Comment:
What about the snapshots? Have we determined what to do with that? Or was this deleted by mistake? 😄

Reply:
Yeah, we won't allow users to manage the snapshots through the configured Cloud APIs, since we're moving towards having the Elasticsearch APIs just do that. I closed #3 to reflect that, so I was just cleaning up the TODOs.